packages/logs/src/logger.ts

```diff
@@ -70,7 +69,7 @@ export function startLogger(errorObservable: ErrorObservable, configuration: Con
             url: window.location.href,
           },
         },
-        (areCookiesAuthorized()) ? { session_id: session.getId() } : {},
+        (session.getId()) ? { session_id: session.getId() } : {},
```

Comment:
you don't need this condition, `undefined` fields are removed during serialization.

```
JSON.stringify({a: undefined})
"{}"
```